Makefile

```diff
@@ -76,6 +76,27 @@ ifeq ($(origin STRINGOPOVERFLOW), undefined)
 	endif
 endif
 
+ifeq ($(origin NOSTRICTOVERFLOW), undefined)
+	NOSTRICTOVERFLOW := $(shell $(CC) -Q --help=warning 2>&1 | grep "strict-overflow" | wc -l)
+	ifneq "$(NOSTRICTOVERFLOW)"  "0"
+	CWFLAGS += -fno-strict-overflow
+	endif
+endif
+
+ifeq ($(origin NODELETENULLPOINTER), undefined)
+	NODELETENULLPOINTER := $(shell $(CC) -Q --help=optimizers 2>&1 | grep "delete-null-pointer-checks" | wc -l)
+	ifneq "$(NODELETENULLPOINTER)"  "0"
+	CWFLAGS += -fno-delete-null-pointer-checks
+	endif
+endif
+
+ifeq ($(origin WRAPV), undefined)
```

Comment:
I don't like how our makefile is expanding. It is right time to start considering automake/cmake.

Reply:
I've started discussion #62